_posts/2019-08-26-how-to-use-gruntwork-service-catalog.adoc

```diff
@@ -0,0 +1,1927 @@
+  fit your exact needs.
+
+Learn best practices::
+  The Service Catalog includes both thorough documentation and access to the
+  https://gruntwork.io/training/[DevOps Training Library], a collection of video training courses that teach a variety
+  of DevOps topics, such as infrastructure as code, Terraform, Docker, Packer, AWS, GCP, security, and more.
+
+Keep everything up to date::
+  We are constantly updating the Gruntwork Service Catalog with the latest best practices, new features, and bug fixes.
+  Instead of spending months fighting with Terraform or Kubernetes updates, better infrastructure is just a
+  version number bump away (see the monthly https://blog.gruntwork.io/tagged/gruntwork-newsletter[Gruntwork Newsletter]).
+
+Get commercial support::
+  Work with a team of DevOps experts who can help you set up your infrastructure, design highly available and
+  scalable systems, automate your builds and deployments, troubleshoot issues, and avoid gotchas and pitfalls. Reach out
+  to Gruntwork Support via Slack, email, and phone/video calls, get code reviews, prioritized bug fixes, and SLAs on
+  response times.
+
+=== What you'll learn in this guide
+
+This guide consists of three main sections:
+
+<<core_concepts>>::
+  An overview of the core concepts you need to understand to use the Gruntwork Service Catalog, including a look into
+  how the Service Catalog is designed, how the Reference Architecture is designed, how we build production-grade
+  infrastructure, and how to make use of infrastructure as code, Terraform, Terragrunt, Packer, Docker, immutable
+  infrastructure, versioning, automated testing, and more.
+
+<<how_to_use_the_catalog>>::
+  A step-by-step walkthrough of how to use the Gruntwork Service Catalog.
+
+<<next_steps>>::
+  What to do next once you've finished reading this guide.
+
+Feel free to read the guide from start to finish or skip around to whatever part interests you!
+
+[[core_concepts]]
+== Core concepts
+
+=== The Gruntwork Service Catalog
+
+// TODO: add a screenshot of the service catalog
+
+The Gruntwork Service Catalog is a collection of over 300,000 lines of reusable, battle-tested infrastructure code.
+that is organized into 40+ GitHub repos, some public and open source, and some private and only accessible to Gruntwork
+customers. Each repo is focused on one type of infrastructure: e.g., there is one repo that contains code for deploying
+and managing Kubernetes on AWS, one repo with code for deploying and managing the ELK (Elasticsearch, Logstash, Kibana)
+stack, one repo that contains a collection of CI / CD code, and so on.
+
+The code in the Gruntwork Service Catalog is written using a combination of:
+
+Terraform::
+  Used to to define and manage most of the basic infrastructure, such as servers, databases, load balancers, and
+  networking.
+
+Go::
+  Used to build cross-platform CLI applications. E.g., We have an app called `ssh-grunt` you can run on each
```

Comment:
And tests! using this super awesome tool also built by us!

Reply:
Looks like a dupe comment. Resolved as per the above: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/151/commits/4603b48a336569db90352172790bc29347291c75